test/files/qty000/pass-qty000-not_activated_psets.ifc

```diff
@@ -0,0 +1,107 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('ViewDefinition [CoordinationView]'),'2;1');
+FILE_NAME('','2024-07-28T20:26:00',(''),(''),'IfcOpenShell-0.7.11-d51fa2c5f','IfcOpenShell-0.7.11-d51fa2c5f','');
+FILE_SCHEMA(('IFC2X3'));
+ENDSEC;
+DATA;
+#1= IFCPERSON('User1\','Undefined',$,$,$,$,$,$);
```

Comment:
```suggestion
#1= IFCPERSON('User1','Undefined',$,$,$,$,$,$);
```
As-is, this test file generates a schema error.

Reply:
After fixing the syntax error and re-validating this file also reports as activated:
![image](https://github.com/user-attachments/assets/34e1a881-5a46-4396-b105-2d1e32058e12)